a10_octavia/controller/worker/controller_worker.py

```diff
@@ -629,6 +636,7 @@ def create_pool(self, pool_id):
         :returns: None
         :raises NoResultFound: Unable to find the object
         """
+        time.sleep(60)
```

Comment:
Don't use sleep(60) but use a general API to check/re-check and wait instead.

Reply:
Fixed